### PR TITLE
memcached back-end

### PR DESCRIPTION
**Pull request by buddisattva**
A paraphrase of redis back-end.

### CODE REVIEW

File: be-memcached.c [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+    }
+    free(query);
+
+    int answer = 0;
+    if (rc == MEMCACHED_SUCCESS) {
+        int x = atoi(value);
+        if (x >= acc)
+            answer = 1;
+    }
+
```
Comment (buddisattva) [inline be-memcached.c]: free(value);

**Comment by jpmens** [comment]
@buddisattva thank you for this contribution: I see you've reviewed your own code; will you push a fix to this PR before I merge it?

**Comment by buddisattva** [comment]
@jpmens thank your reminding. I will push a fix.

**Comment by jpmens** [comment]
Merged, thank you.

**Pull request merged**